ultimatejobweb/models.py

```diff
@@ -13,5 +13,6 @@ class Job(models.Model):
     job_title = models.CharField(max_length=128)
     description_url = models.URLField()
 
-
+    def delete_data():
```

Comment:
This should be a classmethod.
The first parameter should be cls
the code would be 
cls.objects.all().delete()

Reply:
@simzacks What is the purpose of the class?

Reply:
In the way you are using here you should use self,
self means using instance methods.

While the use of @classmethod the 1st parameter is the class itself.
Here when working w/classes inherits from  model and are mapping of db data, we prefer to use  @classmethods working on the class itself

As sim already explained, 
@classmethod
def delete_data(cls):
    cls.objects.all().delete()

**Note:** the same for the push method